From cfb006a3f6eddfa906ed741f927c933a23a1745e Mon Sep 17 00:00:00 2001
From: Alex Deucher <alexdeucher@gmail.com>
Date: Wed, 2 Mar 2011 20:07:38 -0500
Patch-mainline: 2.6.39
References: fate#310916
Git-commit: 3d106fb44e195652ed1c9aac17e833b5302e8bfe
Subject: [PATCH 2189/2588] drm/radeon/kms: additional default context regs
 for cayman

Signed-off-by: Alex Deucher <alexdeucher@gmail.com>
Signed-off-by: Dave Airlie <airlied@redhat.com>
(cherry picked from commit 3d106fb44e195652ed1c9aac17e833b5302e8bfe)

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 drivers/gpu/drm/radeon/cayman_blit_shaders.c |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/drivers/gpu/drm/radeon/cayman_blit_shaders.c b/drivers/gpu/drm/radeon/cayman_blit_shaders.c
index 56be2b6..e148ab0 100644
--- a/drivers/gpu/drm/radeon/cayman_blit_shaders.c
+++ b/drivers/gpu/drm/radeon/cayman_blit_shaders.c
@@ -45,6 +45,11 @@ const u32 cayman_default_state[] =
 	0x00000316,
 	0x0000000e, /* VGT_VERTEX_REUSE_BLOCK_CNTL */
 	0x00000010, /*  */
+
+	0xc0026900,
+	0x000000d9,
+	0x00000000, /* CP_RINGID */
+	0x00000000, /* CP_VMID */
 };
 
 const u32 cayman_default_size = ARRAY_SIZE(cayman_default_state);
-- 
1.7.6

